src/views/ajax.js

```diff
@@ -38,14 +38,14 @@ function update(details) {
         dom.removeClass(counter, 'glimpse-section-value--update');
     }, 2000);
 
-    //manage row values
-    if (state.summaryStack.length === 0) {
-        var section = document.getElementById('glimpse-ajax-summary');
-        section.insertAdjacentHTML('beforeend', '<div class="glimpse-ajax-rows" id="glimpse-ajax-rows"></div>');
-    }
-    recordItem(rowTemplate(details), document.getElementById('glimpse-ajax-rows'), state.summaryStack, 2);
+    state.summaryStack.push(details);
```

Comment:
It looks like we are going to keep pushing items into this array for ever. What `recordItem` used to do was make sure that the array only had what we needed. Also the idea behind `recordItem` was that the same logic can be reused for the popup window.